tests/basics.rs

```diff
@@ -5,53 +5,100 @@ use std::fs::File;
 use std::io::Write;
 use std::path::Path;
 
-use goldenfile::Mint;
+use goldenfile::{Mint, DifferType};
 
-fn setup_file(path: &str, contents: &str) {
-    let path = Path::new(path);
+fn open_file(path: &str) -> File{
+let path = Path::new(path);
```

Comment:
Weird formatting. Run `scripts/install-git-hooks` to automatically run a `cargo fmt` check during commit.